config/includes.chroot/etc/initramfs-tools/scripts/init-premount/loop_mount

```diff
@@ -9,16 +9,117 @@ case "${1}" in
 		;;
 esac
 
+. /scripts/functions
+
 if [ "x${LOOPROOT}" != "x" ]; then
+	_log_msg "Will attempt to use ${LOOPSRC} from ${LOOPROOT} as loop mounted drive"
+
+	lprootp=/mnt/looproot
+	lprootd=/dev/loop0
+
 	sleep 2
 	modprobe loop
-	mkdir -p /mnt/looproot
-	mount -t ntfs-3g ${LOOPROOT} /mnt/looproot
-	losetup /dev/loop0 /mnt/looproot/${LOOPSRC}
-	kpartx -a /dev/loop0
+	mkdir -p ${lprootp}
+
+	log_begin_msg "mounting ${LOOPROOT}"
+	mount -t ntfs-3g ${LOOPROOT} ${lprootp} || {
```

Comment:
Maybe use GNU mount too? Then you will not need to specify fs type, so we will be more flexible in where we can put the image.